frontend/src/pages/NetworkPage/NetworkPage.tsx

```diff
@@ -13,31 +13,39 @@ export const NetworkPage: React.FC = () => {
   const css = useStyles()
   const history = useHistory()
   const { backend } = useDispatch<Dispatch>()
-  const { interfaces, targets, scanData, privateIP } = useSelector((state: ApplicationState) => ({
-    interfaces: state.backend.interfaces,
-    targets: state.backend.targets,
-    scanData: state.backend.scanData,
-    privateIP: state.backend.environment.privateIP,
-  }))
+  const { interfaces, targets, scanData, privateIP, oobAvailable, oobActive } = useSelector(
+    (state: ApplicationState) => ({
+      interfaces: state.backend.interfaces,
+      targets: state.backend.targets,
+      scanData: state.backend.scanData,
+      privateIP: state.backend.environment.privateIP,
+      oobAvailable: state.backend.environment.oobAvailable,
+      oobActive: state.backend.environment.oobActive,
+    })
+  )
 
   const scan = (interfaceName: string) => {
-    analytics.track('networkScan')
     emit('scan', interfaceName)
+    emit('interfaces')
   }
 
   useEffect(() => {
-    emit('interfaces')
+    if (oobAvailable) {
+      emit('interfaces')
+      let timer = setInterval(() => emit('interfaces'), 30000)
```

Comment:
This should be a button, not an interval.

Reply:
I'd prefer to leave this for now.  Also brenda says not to change the UI right now.

Reply:
Can we remove this if we do the heartbeat check please?